project/Dependencies.scala

```diff
@@ -47,7 +47,7 @@ object Dependencies {
   val playGoogleAuth = "com.gu.play-googleauth" %% "play-v28" % "2.1.1"
   val playSecretRotation = "com.gu.play-secret-rotation" %% "play-v28" % "0.18"
   val playSecretRotationAwsSdk = "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v1" % "0.18"
-  val quartzScheduler = "org.quartz-scheduler" % "quartz" % "2.2.3"
+  val quartzScheduler = "org.quartz-scheduler" % "quartz" % "2.3.2"
```

Comment:
Just double checking, isn't this the quartz version that broke admin app the other day? 

revert PR here: https://github.com/guardian/frontend/pull/24909/files

Reply:
Yup - I've tested admin and it seems to be working...

Reply:
Saying this - it's true that the admin app isn't starting up when deploying through riff-raff.

Reply:
This is why it wasn't starting:
https://github.com/guardian/frontend/pull/24962/commits/478fc2c0df68b41264465e0f8eb2b03628d75484

Basically the old version accepted the invalid syntax, there's [an issue](https://github.com/quartz-scheduler/quartz/issues/729) with others having the same problem.

[We should write some tests](https://github.com/guardian/frontend/issues/24964).